### PR TITLE
Remove lost and deprecated data sets

### PR DESCRIPTION
**Pull request by rzumer**
This removes WIP sets, sets missing from AWCY and that cannot be sourced anymore, sets that cannot be redistributed due to licensing terms, and sets that are no longer useful. Most sets that aren't slight variants of others are kept even if they are rarely used.

### CODE REVIEW

File: sets.json [hunk 1]
```diff
@@ -259,82 +152,6 @@
       "wikipedia_420.y4m"
     ]
   },
-  "objective-1-fast-65": {
```
Comment (tdaede) [inline sets.json:262]: We do have this one, I would like to leave it for now as it was historically used several times.

File: sets.json [hunk 2]
```diff
@@ -1125,61 +710,6 @@
       "White_Dune_-_Mui_Ne.y4m"
     ]
   },
-  "subset1-64": {
```
Comment (tdaede) [inline sets.json:1128]: Same for this one.

File: sets.json [hunk 3]
```diff
@@ -3277,145 +2807,5 @@
       "old_town_cross_444_720p50.y4m",
       "park_joy_444_720p50.y4m"
     ]
-  },
-  "scenecuts-720p": {
```
Comment (tdaede) [inline sets.json]: same

**Comment by tdaede** [comment]
LGTM if we keep the above 3 sets. Though I could be convinced to drop them, I'd say the most to least important is scenecuts-720p, subset1-64, and then objective-1-fast-65.

File: sets.json (hunk 3 above)
Comment (rzumer) [reply sets.json]: That set isn't very good, it's mostly the same 2-3 source videos, there are too many clips and not all of them have scene cuts, `vimeo-corpus-10s` was supposed to replace it.

File: sets.json (hunk 1 above)
Comment (rzumer) [reply sets.json:262]: When was it last used and can it be uploaded to https://media.xiph.org/sets/?

File: sets.json (hunk 2 above)
Comment (rzumer) [reply sets.json:1128]: As above

File: sets.json (hunk 3 above)
Comment (tdaede) [reply sets.json]: OK. I'm fine dropping it then.

File: sets.json (hunk 1 above)
Comment (tdaede) [reply sets.json:262]: It was used for AV2 baseline selection (libaom vs SVT-AV1). I can upload it there.

File: sets.json (hunk 2 above)
Comment (tdaede) [reply sets.json:1128]: It was used for debugging frame edge issues on rav1e. I can also upload it.

**Comment by tdaede** [comment]
I have uploaded both to media.xiph.org/sets.

**Comment by rzumer** [comment]
Seems like squashing is disabled for the repository but that would be the best option for merging since solving conflicts now would be a bit annoying.

**Comment by tdaede** [comment]
Can you just locally squash the top commits together with rebase -i?

**Comment by rzumer** [comment]
I tried and it made a huge mess with the history, I could try again tomorrow.

**Comment by vibhoothi** [comment]
Superseded with https://github.com/xiph/rd_tool/pull/162

**Pull request closed without merging**